huaweicloud/services/acceptance/cdn/resource_huaweicloud_cdn_domain_batch_copy_test.go

```diff
@@ -0,0 +1,148 @@
+package cdn
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccCdnDomainBatchCopy_basic(t *testing.T) {
```

Comment:
```suggestion
func TestAccDomainBatchCopy_basic(t *testing.T) {
```
Remove the package name.

Reply:
done